g_docs.go

```diff
@@ -89,7 +89,9 @@ func generateDocs(curpath string) {
 				} else if strings.HasPrefix(s, "@Title") {
 					rootapi.Infos.Title = strings.TrimSpace(s[len("@Title"):])
 				} else if strings.HasPrefix(s, "@Description") {
-					rootapi.Infos.Description = strings.TrimSpace(s[len("@Description"):])
+					desc := strings.TrimSpace(s[len("@Description"):])
+					desc = strings.ReplaceAll(desc, "\\n", "\n")
```

Comment:
why do we need this? I tested in Doraemon repo but the swagger files generated with/without this line are identical

Reply:
ah sorry I skipped the PR description 🙈